collector/rpc_metrics.go

```diff
@@ -98,6 +113,30 @@ func NewNodeRpcMetrics(
 			nil,
 			nil,
 		),
+		internalStatusRPCFail: prometheus.NewDesc(
+			"near_internal_status_rpc_fail",
+			"The number of failed internal RPC calls.",
+			nil,
+			nil,
+		),
+		internalStatusRPCSuccess: prometheus.NewDesc(
+			"near_internal_status_rpc_success",
+			"The number of successful internal RPC calls.",
+			nil,
+			nil,
+		),
+		externalStatusRPCFail: prometheus.NewDesc(
+			"near_external_status_rpc_fail",
+			"The number of failed external RPC calls.",
+			nil,
+			nil,
+		),
+		externalStatusRPCSuccess: prometheus.NewDesc(
+			"near_external_status_rpc_success",
+			"The number of successful external RPC calls.",
+			nil,
+			nil,
+		),
```

Comment:
Could also potentially do:
```
		internalStatusRPCFail: prometheus.NewDesc(
			"near_internal_status_rpc_calls",
			"The number of internal RPC calls.",
			nil,
			nil,
		),
		internalStatusRPCSuccess: prometheus.NewDesc(
			"near_internal_status_rpc_success",
			"The number of successful internal RPC calls.",
			nil,
			nil,
		),
```

So instead of reporting success and failures, you can just report successes + total attempts.
You could then derive your number of failures by doing total attempts - successes, and this might simplify some surrounding logic within these exporter files - like within https://github.com/bisontrails/near-prometheus-exporter/pull/8/files#diff-8a12b10fd6c7a3a22b6febab64f0d0a2bc8a75a3358d50e83594490f4f9ac010R306-R328 for example

Though you'd also be adding more prometheus logic downstream to boot, so this is very debatable.

Reply:
Meh. `Total` isnt as useful as direct `fails` and `successes`. Total can be infirmed from `fails` + `successes` if for some reason you need the total rpc queries